src/score_components/find-problematic-comments.js

```diff
@@ -34,6 +37,7 @@ module.exports = function grepForProblematicComments(ext, include, exclude) {
     output = child_process.execSync(find).toString().trim();
   } catch (e) {
     // TODO: handle error
+    output = ''; // temporary fix to avoid undefined
```

Comment:
We probably want to extend this function so that it accepts the `core` parameter, just like `coverage` does, and use that to call `core.error` to report the error encountered here, otherwise this will fail silently.

Or, we just file that as an issue to deal with in the future.

Reply:
Will fix this while adding the annotations (PR #23)